packages/markers/src/browser/markers-manager.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable } from "inversify";
+import URI from "@theia/core/lib/common/uri";
+import { Disposable, Event, Emitter } from "@theia/core/lib/common";
+
+export interface Marker {
+    uri: URI;
```

Comment:
This should be a `string`, isn't it? If so, probably `MarkerFile` has to be adjusted too.